streaming/streaming.go

```diff
@@ -61,12 +58,15 @@ type DatastreamCfg struct {
 }
 
 type DataflowCfg struct {
-	JobName            string
-	Location           string
-	HostProjectId      string
-	Network            string
-	Subnetwork         string
-	DbNameToShardIdMap map[string]string
+	JobName             string
+	Location            string
+	HostProjectId       string
+	Network             string
+	Subnetwork          string
+	MaxWorkers          string
```

Comment:
why is this a string instead of an integer ?

Reply:
It is easier to handle unspecified cases (via '') in string as opposed to treating 0 as unspecified in ints. 
Plus for the CLI, the streaming cfg is manually specified, where the user can leave unspecified with string as opposed to providing 0 for integer.

We can use ints as well but it will require some additional code changes to cover CLI and UI both.